inji-web/src/__tests__/utils/i18n.test.tsx

```diff
@@ -0,0 +1,159 @@
+
+import { initReactI18next } from 'react-i18next';
+import { setReduxState, clearReduxState } from '../../test-utils/reduxMockUtils';
+import en from '../../locales/en.json';
+import fr from '../../locales/fr.json';
+import ta from '../../locales/ta.json';
+import hi from '../../locales/hi.json';
+import kn from '../../locales/kn.json';
+import ar from '../../locales/ar.json';
+
+// Mock i18next
+const mockI18n = {
+  use: jest.fn().mockReturnThis(),
+  init: jest.fn().mockResolvedValue(undefined),
+  changeLanguage: jest.fn().mockResolvedValue(undefined),
+  language: 'en',
+  t: jest.fn((key: string) => key),
+};
+
+jest.mock('i18next', () => mockI18n);
+
+// Mock react-i18next
+jest.mock('react-i18next', () => ({
+  initReactI18next: {
+    type: '3rdParty',
+    init: jest.fn(),
+  },
+  useTranslation: () => ({
+    t: (key: string) => key,
+    i18n: mockI18n,
+  }),
+}));
+
+// Mock storage
+const mockStorage = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  SELECTED_LANGUAGE: 'selectedLanguage',
+};
+
+jest.mock('../../utils/storage', () => ({
+  storage: mockStorage,
+}));
+
+// Mock window
+const windowSpy = jest.spyOn(global, 'window', 'get');
+
+describe('i18n configuration', () => {
+  let i18nModule: any;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    windowSpy.mockClear();
+    clearReduxState();
+    mockStorage.getItem.mockReset();
+    mockStorage.setItem.mockReset();
+
+    windowSpy.mockImplementation(() => ({
+      _env_: {
+        DEFAULT_LANG: 'en',
+      },
+    } as Window & typeof globalThis));
+
+    setReduxState({
+      language: 'en',
+    });
+
+    jest.isolateModules(() => {
+      i18nModule = jest.requireActual('../../utils/i18n');
+    });
+  });
+
+  afterEach(() => {
+    jest.resetModules();
+    windowSpy.mockRestore();
+    clearReduxState();
+  });
+
+  describe('initialization', () => {
```

Comment:
use standard naming for Describe and Test

Describe usually starts with "Test*"
Describe usually starts with "Check*"